docs/installation/autoware/docker-installation.md

```diff
@@ -16,21 +16,137 @@
    git clone https://github.com/autowarefoundation/autoware.git
    ```
 
-2. Install the dependencies.
+2. You can install the dependencies either manually or using the provided Ansible script.
 
-   ```bash
-   ./setup-dev-env.sh docker
-   ```
+### Installing dependencies manually
```

Comment:
Same comment as https://github.com/autowarefoundation/autoware-documentation/pull/63#issuecomment-1079606375 and https://github.com/autowarefoundation/autoware-documentation/pull/63#issuecomment-1079609092.

I think this enables users to skip unnecessary steps that they have already installed.

Reply:
Done!